docs/getting-started-guides/README.md

```diff
@@ -8,6 +8,7 @@ GKE            |              |        | GCE         | [docs](https://cloud.goog
 AWS            | CoreOS       | CoreOS | flannel     | [docs](../../docs/getting-started-guides/coreos.md)    | Community                    | Uses K8s version 0.11.0
 GCE            | CoreOS       | CoreOS | flannel     | [docs](../../docs/getting-started-guides/coreos.md)    | Community (@kelseyhightower) | Uses K8s version 0.11.0
 Vagrant        | CoreOS       | CoreOS | flannel     | [docs](../../docs/getting-started-guides/coreos.md)    | Community (@pires)           | Uses K8s version 0.11.0
+Vagrant        | CoreOS       | CoreOS | flannel     | [docs](https://github.com/AntonioMeireles/kubernetes-vagrant-coreos-cluster/blob/master/README.md)    | Community (@AntonioMeireles)           | Uses K8s version 0.13.2
```

Comment:
Azure, a few lines bellow that opened the precedent for out of this tree repos...  i 'm still think that my approach may be a bit too much for being inline (see my comments above) in the core source tree. OTOH it's handy to have available pointers for users who want to go deeper. [and avoiding one less hit to Google/the search engine]